tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -11,15 +11,15 @@ next-page: classes
 previous-page: basics
 ---
 
-In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates the class hierarchy.
+In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates a subset the class hierarchy.
 
-![Scala Type Hierarchy]({{ site.baseurl }}/resources/images/classhierarchy.img_assist_custom.png)
+![Scala Type Hierarchy]({{ site.baseurl }}/tutorial/tour/unified-types-diagram.svg)
 
 ## Scala Class Hierarchy ##
 
-The superclass of all classes `scala.Any` has two direct subclasses: `scala.AnyVal` and `scala.AnyRef`.
+`scala.Any` is the base type of all types. It defines certain universal methods: `==`, `!=`, `equals`, `hashCode`, `toString`. `scala.Any` has two direct subclasses: `scala.AnyVal` and `scala.AnyRef`.
 
-`scala.AnyVal` represents value classes. All value classes are non-nullable and predefined; they correspond to the primitive types of Java-like languages. Note that the diagram above also shows implicit conversions between the value classes.
+`scala.AnyVal` represents value classes. All value classes are non-nullable and predefined; they correspond to the primitive types of Java-like languages: `Byte`, `Short`, `Char`, `Int`, `Long`, `Float`, `Double`, and `Boolean`. Scala adds the `Unit` type which is similar to Java's `void`.
```

Comment:
The automatic conversions between primitives is really important to know about and shouldn't be left out.  `Unit` is not similar to `void` in a very important way which is that it is a value that you can return, unlike `void` which is a special case of not returning anything; if that isn't part of the tour by this point, you must explain it or "similar to void" is more misleading than helpful.